ParkingFinder/repositories/real_time_repository.py

```diff
@@ -0,0 +1,97 @@
+from sqlalchemy.orm.exc import NoResultFound
+from tornado.gen import coroutine, Return
+
+from ParkingFinder.base.async_db import create_session
+from ParkingFinder.base.errors import NotFound
+from ParkingFinder.mappers.real_time_mapper import RealTimeMapper
+from ParkingFinder.tables.real_time import RealTimes
+
+
+class RealTimeRepository(object):
+
+    @staticmethod
+    @coroutine
+    def read_one(user_id):
+        """
+        Read from realtime with user_id 
+
+        :param str key:
+        :return data:
+        :raises sqlalchemy.orm.exc.NoResultFound:
+        """
+        with create_session() as session:
+            realtime = session.query(RealTimes).filter(
+                RealTimes.waiting_user_id == user_id
```

Comment:
I think waiting_use_id == user_id. Or request user id == user id might be better at this point. Make the function as generic as possible.